src/benchmarks/micro/libraries/System.Collections/TryGetValue/TryGetValueFalse.cs

```diff
@@ -15,6 +15,7 @@ namespace System.Collections
 {
     [BenchmarkCategory(Categories.Libraries, Categories.Collections, Categories.GenericCollections)]
     [GenericTypeArguments(typeof(int), typeof(int))] // value type
+    [GenericTypeArguments(typeof(SmallClass), typeof(SmallClass))] // reference type
```

Comment:
It would be nice if we could add a few large structs as well, tuples are fairly common dictionary keys.

Reply:
Done